README.md

```diff
@@ -1 +1,1181 @@
-# ballerina-honeycomb
\ No newline at end of file
+  [![Build Status](https://travis-ci.org/ballerina-guides/ballerina-honeycomb)](https://travis-ci.org/ballerina-guides/ballerina-honeycomb)
+  
+## Integration with Honeycomb
+
+Honeycomb is a tool used to investigate how well your system works in various conditions (for example - high traffic). Through Honeycomb, you can collect data pertaining to your software that can be broken down into various entities. You can observe the performance of each of these entities specifically.
+     
+> This guide provides instructions on how Ballerina can be used to integrate with Honeycomb.
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-you’ll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Configuration with Honeycomb](#testing-with-honeycomb)
+     - [Traces](#views-of-traces)
+     - [Metrics](#metrics)
+
+## What you’ll build
+To perform this integration with Honeycomb,  a real world use case of a very simple student management system is used.
+This system will illustrate the manipulation of student details in a school/college management system. The administrator will be able to perform the following actions in this service.
+
+    - Add a student's details to the system.
+    - List down all the students' details who are registered in the system.
+    - Delete a student's details from the system by providing student ID.
+    - Generate a mock error (for observability purposes).
+    - Get a student's marks list by providing student ID.
+
+![Honeycomb](images/ballerina-honeycomb.svg "Ballerina-Honeycomb")
+
+- **Make Requests** : To perform actions on student  management service, a console based client program has been written in Ballerina for your ease of making requests.
```

Comment:
There is an extra space between "student" and "management". Also, let's add "the" before "student management service". 

Change "console based" to "console-based". Normally when two words are linked closely together like this we use a hyphen.